libgnucash/app-utils/gnc-optiondb.i

```diff
@@ -0,0 +1,1569 @@
+/*
+ * Temporary swig interface file while developing C++ options.
```

Comment:
Is it still temporary or does that refer to your PR comment you want to integrate this into swig-engine ?

Reply:
When I wrote that I was thinking of rolling it into app-utils.i or engine.i, but it's big enough to keep as a separate file so I guess it's no longer temporary.

Reply:
Dunno about luckily, the behavior is needed for scheme serialization. But that's tested in scheme from test-gnc-option-scheme-output, so it indeed doesn't need to be here.